platform-cloud/docs/studios/add-studio.md

```diff
@@ -0,0 +1,84 @@
+
+:::info
+Write permissions can be added for specific AWS S3 buckets in the compute environment configuration by defining additional **Allowed S3 Buckets** during creation. This means that data can be written from the session back to the cloud storage path(s) mounted. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the session immediately.
+:::
+
+## General config
+
+### Seqera-managed container template images
+
+You can customize the following fields:
+- **Container template**: Select a template from the dropdown list.
+- **Studio name**
+- Optional: **Description**
+
+Select **Install Conda packages** to define or upload a list of Conda packages to include with the Studio. For more information on the syntax for specifying Conda packages, see [Conda package syntax][conda-syntax].
+
+**Collaboration**: By default, all Studios are collaborative. This means all workspace users with the launch role and above can connect to the session. You can toggle **Private** on which means that only the workspace user who created the Studio can connect to it. When **Private** is on, workspace administrators can still start, stop, and delete sessions but cannot connect to them.
+
+**Session lifespan**: Depending on your workspace settings, you may be able to choose between the following options:
+
+- **Stop the session automatically after a predefined period of time:** If there is an existing defined session lifespan workspace setting, you won't be able to edit this. If no workspace setting is defined, you can edit this field. The minimum value is 1 hour and the maximum is 120 hours. The default value is 8 hours. If you change the default value, the change applies only to that session. Once you've stopped the session, the value returns to default.
+- **Keep the session running:** The session will stay active (and consuming compute resources) until it's manually stopped or encounters an error which ends the session.**
+
+#### Environment variables
+
+For the selected compute environment, all existing environment variable key-value pairs are automatically inherited and displayed for the studio session. Additional per-session environment variables can be defined for the Studio.
+     
+Session-level environment variables take precedence. You can overwrite an existing compute environment-defined environment variable key-value pair by defining the same key and a different value.
+
+### Custom container template image (BYOC)
+
+Customize the following fields:
+ **Container template**: Select **Prebuilt container image** from the list. For information about providing your own template, see [Custom container template image][custom-image].
+ If you select the **Prebuilt container image** template, you cannot select **Install Conda packages** as these options are currently mutually exclusive.
+ - **Studio name**
+- Optional: **Description**
```

Comment:
```suggestion
```
I felt like this would be better added into the main flow above as a choice you can make.